cobra/flux_analysis/sampling.py

```diff
@@ -496,8 +592,7 @@ def __getstate__(self):
         return d
 
 
-def sample(model, n, method="optgp", processes=1, seed=None,
-           solver=None, **solver_kwargs):
+def sample(model, n, method="optgp", processes=1, seed=None):
```

Comment:
Yeah, major run time actually comes from getting the initial search directions. Thought about alternatives but it's difficult since the initial vectors need to span all dimensions. It comes back to the known conundrum of getting the fluxes fast which is the major bottleneck here. tqdm could be nice but in that case we would also want it for other functions such as double deletions, flux variability etc...